.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing to Haaukins 
+
+If you wish to make contribution to Haaukins platform, you are very welcome to do it, any contribution would be beneficial. 
+
+## Pull requests are always welcome
+
+We are always thrilled to receive pull requests, and do our best to process them as quickly as possible. Not sure if that typo is worth a pull request? Do it! We will appreciate it.
+
+If your pull request is not accepted on the first try, don't be discouraged! If there's a problem with the implementation, hopefully you received feedback on what to improve.
+
+## Conventions 
+
+Fork the repo and make your changes on your fork in a feature branch. Haaukins developments is based on [Git Feature Branch Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
+
+- For bug fixes and feature requests, the name of branch that you will work on should follow the convention [XXXX]-something, XXXX refers to issue number. 
+
+- There should be a naming convention (function names and variable names should follow defined pattern such as getName(private) or GetName (public).
+
+- Having clean formatted codes are always easy to review, hence the changes on branch require to have `gofmt` to format the code. 
```

Comment:
- Use `gofmt` to ensure code formatting that is easy to read.

Reply:
... a suggestion for a more concise wording ("Rule"  comes first, followed by reasoning)